frontend/device/android/device.lua

```diff
@@ -375,6 +377,21 @@ function Device:canExecuteScript(file)
     end
 end
 
+function Device:lightDialog()
+    local usleep = require("ffi/util").usleep
+    local title = android.isEink() and _("Frontlight settings") or _("Light settings")
+    android.settings.dialog(title, _("Brightness"), _("Warmth"), _("Ok"))
```

Comment:
Should be OK or something more descriptive.

See http://koreader.rocks/doc/modules/ui.widget.confirmbox.html

Reply:
It is the button that allows you to exit the dialog. See https://github.com/koreader/koreader/issues/5768#issuecomment-652443499

Reply:
For a single button it might be better to go with "Close" than "OK"?

Better UX might be "Cancel" and "OK," which is also what's on the dialog my phone pops up from settings. (Which I've probably never used because the dropdown from the top is much more relevant.)

Reply:
> For a single button it might be better to go with "Close" than "OK"?

Both are fine for me :)

> Better UX might be "Cancel" and "OK," which is also what's on the dialog my phone pops up from settings.

But cancel does the same as ok/close. Changes are already applied.

A cancel button would mean discard any change done inside the dialog and revert to previous values. It is doable and can be done quite fast if the change is desiderable.

Reply:
> A cancel button would mean discard any change done inside the dialog and revert to previous values.

Exactly. ;-)

>  if the change is desiderable.

It seems to be in line with how Android works, unless they've changed it in recent versions.

Reply:
ok, done in the backend. Also updated the diff at the end of this PR.

Reply:
I think you still need to push it? :+1: